package.json

```diff
@@ -81,12 +81,14 @@
     "@codemirror/search": "^6.5.6",
     "@codemirror/state": "^6.4.1",
     "@codemirror/view": "^6.26.0",
+    "@cortex-js/compute-engine": "^0.25.1",
     "@ffmpeg/core": "^0.11.0",
     "@ffmpeg/ffmpeg": "^0.11.6",
     "@fluent/bundle": "^0.18.0",
     "@lezer/common": "^1.2.1",
     "@lezer/lr": "^1.4.0",
     "client-zip": "^2.4.4",
+    "compute-engine-to-desmos-latex": "^0.0.1",
```

Comment:
It's better to bring this package directly into the repository (which is a monorepo). Otherwise, development is a dance (`npm link` and stuff is tricky), and updates to the package would require many more steps (make the package change, release it to npm, update the version in desmodder, etc.)

If desired, we could still publish it (and it could even be in the `@desmodder` namespace), but I suspect there's not much demand for it.

Reply:
I'm using it in several other projects of mine, that's why I was publishing it. In fact, I nearly put it in a monorepo for another project. But if it can be put in here and still be published on `@desmodder`, that'd be fine.